src/common/webpack/config.ts

```diff
@@ -461,7 +463,70 @@ function createJavaScriptLoader({
     config,
     isSsr,
 }: HelperOptions): webpack.RuleSetUseItem {
+    if (config.javaScriptLoader === 'swc') {
+        if (config.bundler !== 'rspack') {
+            throw new Error('SWC is supported only with rspack bundler');
+        }
+
+        const browserslist = require('browserslist');
+
+        const options: SwcLoaderOptions = {
+            module: {
+                type: 'es6',
+            },
+            env: {
+                targets: isSsr ? {node: 'current'} : browserslist(),
```

Comment:
swc does not support 'current'.
```suggestion
                targets: isSsr ? {node: process.versions.node} : browserslist(),
```